### DISCUSSION TITLE
Compose new dashboard from existing dashboard

### DISCUSSION THREAD

**Discussion by vishiy** [grafana/grafana#42411] [Dashboards, Panels & Visualizations]
I am trying to compose a new dashboard from an existing dashboard and its panels. Is this possible thru grafonnet APIs ?

Can i do something like below -

local apiserverdashboard = import "apiserver.json";

grafana.dashboard.new("my custom dashboard").addPanels(apiserverdashboard.panels["myapiserverdashboard-panel1"]); ?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
